shared/dev/dumb-sheet/render.desktop.js

```diff
@@ -34,11 +56,29 @@ class Render extends Component<void, Props, any> {
 
   render () {
     const parts = this.props.dumbFilter.toLowerCase().split(':')
-    const filter = parts[0]
-    let numItemsLeftWeCanShow = parseInt(parts[1], 10) || 10
+    let numItemsLeftWeCanShow = 10
+    let filter = parts.join(':')
+
+    if (parts.length > 1) {
```

Comment:
this actually fixes a bug where if you have a key with `:` in it bad stuff will happen. Instead we see if there is a `:{number}` then merge the text so it all works